### PR TITLE
Add RN for removal self link in managedObject parents fragment in res…

### PR DESCRIPTION
**Pull request by pkepa-c8y**
…ponse

### CODE REVIEW

**Review by jaro-herod — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-10-08
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md]:
```suggestion
date: 
```

File: content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: Removed Self link from inventory parents fragment in API response
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md]:
```suggestion
title: Removed self link from parents fragments in Inventory API responses
```

File: content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-JlFdtOPva
+    label: Rest API
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-60938
+version: 10.20.589.0
+---
+The <code>self</code> field has been removed from inventory API from deviceParents, assetParents and additionParents response fragments, it was not required and always had a generated link to a non-existent endpoint.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md]:
```suggestion
In the Inventory API, the `self` field has been removed from `deviceParents`, `assetParents` and `additionParents` response fragments. It was not required and always had a generated link to a non-existent endpoint.
```

File: content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md (hunk 1 above)
Comment (pkepa-c8y) [reply content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md]: Changed.

File: content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md (hunk 2 above)
Comment (pkepa-c8y) [reply content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md]: Changed.

File: content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md (hunk 3 above)
Comment (pkepa-c8y) [reply content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md]: Changed.

**Review by BeateRixen — APPROVED** [review]

**Review by pink-mongoose — COMMENTED** [review]
is this a breaking change?

I would suggest adding in the reason why we have removed it, even if we think it is obvious.  Perhaps: In order to simplify the Inventory API the redundant 'self' field has been removed from ....

File: content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-JlFdtOPva
+    label: Rest API
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-60938
+version: 10.20.589.0
+---
+In the Inventory API, the `self` field has been removed from `deviceParents`, `assetParents` and `additionParents` response fragments. It was not required and always had a generated link to a non-existent endpoint.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md]:
```suggestion
In order to simplify the Inventory API the redundant `self` field has been removed from `deviceParents`, `assetParents` and `additionParents` response fragments. It was not required and always had a generated link to a non-existent endpoint.
```

**Comment by BeateRixen** [comment]
> is this a breaking change?
> 
> I would suggest adding in the reason why we have removed it, even if we think it is obvious. Perhaps: In order to simplify the Inventory API the redundant 'self' field has been removed from ....

Sounds good. Changed the text accordingly.

**Pull request merged**